p01-contact/lang/it.yml

```diff
@@ -1,13 +1,12 @@
 ---
 # Italian language file for p01-contact
 key: it
-name: Italian
+name: Italiano
 english_name: Italian
-authors: Nicola Sarti
-date: "2011-10-27"
+authors: Nicola Scattaglia
```

Comment:
You should add yourself to the existing authors, and feel free to add a contact mail. For example : 
```yml
authors: Nicola Sarti, Nicola Scattaglia <contact@domain.tld>
```

Reply:
will do, thank you!